docs/INSTALL-cloud.md

```diff
@@ -1,6 +1,6 @@
 **Set up Discourse in the cloud in under 30 minutes** with zero knowledge of Rails or Linux shell. One example is [DigitalOcean][do], but these steps will work on any **Docker-compatible** cloud provider or local server.
 
->  🔔 Don't have 30 minutes to set this up? For a flat one-time fee of $150, the community can install Discourse in the cloud for you. [Click here to purchase a self-supported community install](https://www.literatecomputing.com/product/discourse-install/).
+>  🔔 Don't have 30 minutes to set this up? For a one time fee as low as $150 or less, the community can install Discourse in the cloud for you. [Click here to purchase a self-supported community install](https://meta.discourse.org/new-topic?title=%5BPAID%5D+Discourse+Installation&body=I+would+like+to+purchase+a+one-time+community+install&category=marketplace).
```

Comment:
@osioke let's keep Jay in there for now, unless we hear from Jeff about it. That would also remove the need for the `or less` bit 👍